sympy/sets/fancysets.py

```diff
@@ -1338,6 +1345,8 @@ def _contains(self, other):
         isTuple = isinstance(other, Tuple)
         if isTuple and len(other) != 2:
             raise ValueError('expecting Tuple of length 2')
+        if other is S.ComplexInfinity:
+            return False
```

Comment:
Thanks @jksuom . I added lines for this in [new commit](https://github.com/sympy/sympy/pull/11409/commits/28dfa78c88e39455981860922c81459b802f1ce7).